taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from taxi.models import Driver, Car
+from django.contrib.auth import get_user_model
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
```

Comment:
Ensure that the `license_number` field in `DriverLicenseUpdateForm` is validated to meet the specified format: 3 uppercase letters followed by 5 digits. This is crucial for both creation and updating of drivers as per the task requirements.